tes/models.py

```diff
@@ -10,22 +10,18 @@
 from typing import Any, Dict, List, Optional, Tuple, Type, Union
 
 
-@attrs
+@attrs(repr=False)
 class _ListOfValidator(object):
     type: Type = attrib()
 
     def __call__(self, inst, attr, value):
-        """
-        We use a callable class to be able to change the ``__repr__``.
-        """
         if not all([isinstance(n, self.type) for n in value]):
             raise TypeError(
-                "'{attr.name}' must be a list of {self.type!r} (got {value!r} "
-                "that is a list of {values[0].__class__!r}).",
-                attr, self.type, value,
+                f"'{attr.name}' must be a list of {self.type!r} (got "
+                f"{value!r}", attr
             )
 
-    def __repr__(self) -> str:
+    def __repr__(self):
         return f"<instance_of validator for type {self.type!r}>"
```

Comment:
This one took me a while: I couldn't figure out why the `__repr__` never got called in the unit tests and instead `repr()` on the validator always returned `_ListOfValidator(type=<class 'tes.models.Input'>)` or some such. Turns out that `attrs` overrides `__repr__` unless you explicitly tell it not to.  Probably due to a change in `attrs` that came up now that we upgraded dependencies. Setting `repr` to `False` in the decorator fixed that.